src/main/java/core/basesyntax/util/ColorSupplier.java

```diff
@@ -0,0 +1,12 @@
+package core.basesyntax.util;
+
+import core.basesyntax.model.Color;
+import java.util.Random;
+
+public class ColorSupplier {
+    private static final Random random = new Random();
+
+    public String getRandomColor() {
+        return Color.values()[random.nextInt(Color.values().length)].toString();
```

Comment:
![image](https://github.com/mate-academy/jv-lottery/assets/170757042/a59216ef-dd36-4992-98e5-bcf60791de96)

Reply:
Not fixed